src/achievements.config.ts

```diff
@@ -160,12 +160,12 @@ export default AchievementSpec({
     ],
   },
   'houston-bot-merges': {
-    repo: 'houston-bot',
+    repo: 'houston-discord',
     stat: 'merges',
     achievements: [
-      { count: 1, title: 'Assistant', details: 'First houston-bot PR' },
-      { count: 5, title: 'Botanist', details: '5 houston-bot PRs' },
-      { count: 15, title: 'Chatterbox', details: '15 houston-bot PRs' },
+      { count: 1, title: 'Assistant', details: 'First Discord Bot PR' },
+      { count: 5, title: 'Botanist', details: '5 Discord Bot PRs' },
+      { count: 15, title: 'Chatterbox', details: '15 Discord Bot PRs' },
```

Comment:
We’ve kind of standardised on these matching the repo name (helps people know which repo this is for), so although a little less pretty, we should probably stick to that here too:
```suggestion
      { count: 1, title: 'Assistant', details: 'First houston-discord PR' },
      { count: 5, title: 'Botanist', details: '5 houston-discord PRs' },
      { count: 15, title: 'Chatterbox', details: '15 houston-discord PRs' },
```